sonar-python-plugin/src/main/java/org/sonar/plugins/python/IPynb.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.python;
+
+import org.sonar.api.resources.AbstractLanguage;
+
+public class IPynb extends AbstractLanguage {
+
+  public static final String KEY = "ipynb";
+
+  private static final String[] DEFAULT_FILE_SUFFIXES = { ".ipynb" };
+
+  public IPynb() {
+    super(KEY, "IPynb");
+  }
+
+  @Override
+  public String[] getFileSuffixes() {
```

Comment:
Should we allow for custom file suffixes like for Python?
In my opinion, no: since we expect to receive files that were pre-processed by SonarLint, we probably want to avoid dealing with unexpected files that may have bypassed SonarLint processing.